src/frontend/main/assets/MenuIcon.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const MenuIcon = () => {
+  return (
+    <svg
+      focusable="false"
+      aria-hidden="true"
+      width="24"
+      height="18"
+      viewBox="0 0 24 18"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M1.5 3H22.5C23.325 3 24 2.325 24 1.5C24 0.675 23.325 0 22.5 0H1.5C0.675 0 0 0.675 0 1.5C0 2.325 0.675 3 1.5 3Z"
+        fill="black"
+      />
+      <path
+        d="M22.5 7.5H1.5C0.675 7.5 0 8.175 0 9C0 9.825 0.675 10.5 1.5 10.5H22.5C23.325 10.5 24 9.825 24 9C24 8.175 23.325 7.5 22.5 7.5Z"
+        fill="black"
+      />
+      <path
+        d="M22.5 15H1.5C0.675 15 0 15.675 0 16.5C0 17.325 0.675 18 1.5 18H22.5C23.325 18 24 17.325 24 16.5C24 15.675 23.325 15 22.5 15Z"
+        fill="black"
+      />
```

Comment:
I wonder if this needs to be duplicated here..? Would it be possible to just `import` the SVG file (`menu-icon.svg`) here, and let webpack bundle it?

If it's a big hassle then not worth it of course!

Reply:
I did it this way for some icons for which the fill had to be to be changed dynamically, but I guess this one wouldn't really need it (unless we add hover styles or something). Also for accessibility `focusable="false"` and `aria-hidden="true"` should be added for decorative svg icons, but those could be added in the svg file directly. 
So if I clean up the svgs, maybe we can use this format only for the ones that need the fill changed and normal svg for others. And then probably the icons that have this format don't need the separate svg file 🤔But this could be a separate pr, if I change all at the same time?

Reply:
Yeah, and unless we have a LOT of these, I'm not sure it's worth it anyway. Just spotted some repetitive-looking and brought it up. :smile: